docs/team/kengxiii.md

```diff
@@ -1,46 +1,66 @@
 
-* **Enhancements to existing features**:
-  * Updated the GUI color scheme (Pull requests [\#33](), [\#34]())
-  * Wrote additional tests for existing features to increase coverage from 88% to 92% (Pull requests [\#36](), [\#38]())
+* **Enhancements to existing features**: Modified the `find` command to work like a filter.
+  * What it does: `Find` command supports keyword search for `name`, `level`, `subjects` and `overdue` status.
+  * Justification: The previous `find` function returns matched `tutee` whose name fulfils any of the keyword supplied.
+    However, we felt that the feature is not very helpful for tutors if we wanted to extend it to support `level`,
+    `subject` and `overdue` status as well.<br><br>
+    For example, if the tutor wants to search up all secondary 4 students taking math classes, using
+    `find l/p4 subject/math` will return tutees who are **either** primary 4 or tutee's taking math classes.
+    The tutor would then have to sieve through the results once again to find the tutee of interest.<br><br>
+    Hence, we modified the implementation to work just like a filter placed sequentially: a matched tutee has to pass all
+    the filters to be returned as a matched tutee.
+  * Highlights: This enhancement did not affect existing commands and commands to be added in future, but the old
+    test cases using the old implementation had to be modified to align with our new design of the `find` command.
+  * Credits: *-*
 
 * **Documentation**:
   * User Guide:
-    * Added documentation for the features `delete` and `find` [\#72]()
-    * Did cosmetic tweaks to existing documentation of features `clear`, `exit`: [\#74]()
+    * Added documentation for the features `deleteLesson` and `find`.
   * Developer Guide:
-    * Added implementation details of the `delete` feature.
+    * Added implementation details of the `find` feature.
 
 * **Community**:
-  * PRs reviewed (with non-trivial review comments): [\#12](), [\#32](), [\#19](), [\#42]()
-  * Contributed to forum discussions (examples: [1](), [2](), [3](), [4]())
-  * Reported bugs and suggestions for other teams in the class (examples: [1](), [2](), [3]())
-  * Some parts of the history feature I added was adopted by several other class mates ([1](), [2]())
-
-* **Tools**:
-  * Integrated a third party library (Natty) to the project ([\#42]())
-  * Integrated a new Github plugin (CircleCI) to the team repo
-
-* _{you can add/remove categories in the list above}_
+* PRs reviewed (with non-trivial review comments):
+[\#39](https://github.com/AY2122S1-CS2103T-F12-3/tp/pull/39),
+[\#74](https://github.com/AY2122S1-CS2103T-F12-3/tp/pull/74),
+[\#155](https://github.com/AY2122S1-CS2103T-F12-3/tp/pull/155),
+[\#159](https://github.com/AY2122S1-CS2103T-F12-3/tp/pull/159),
+* Took and cropped profile photos for [About-Us](https://ay2122s1-cs2103t-f12-3.github.io/tp/AboutUs.html) page.
```

Comment:
Maybe this portion can put under `Project Management` portion, and talk about setting up the aboutUs page

Reply:
I took the suggestion and moved it to `Project Management` :)